### DISCUSSION TITLE
Introducing a new blog section

### DISCUSSION THREAD

**Discussion by blapie** [shibatch/sleef#583] [Show and tell]
Hello everyone!

We have just introduced [a new blog section on the SLEEF project website](https://sleef.org/7-blog/).
Our [first post](https://sleef.org/2024/10/02/new-pulse.html) came out today and summarises a year of efforts from the community and maintainers to revive SLEEF.

Hope you enjoy the read!

**Comment by shibatch** [comment]
The link to Contribution Guidelines seems to be broken.

**Reply by blapie** [reply]
Thank you, should be fixed.
